src/main/java/mate/academy/springboot/swagger/util/SortUtilImpl.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortUtilImpl implements SortUtil {
```

Comment:
```suggestion
@Component
public class SortServiceImpl implements SortService {
```

It looks strange to have util as spring component